app/views/nominations/choose_location.html.erb

```diff
@@ -0,0 +1,26 @@
+<% content_for :title, "Resend nomination: search location" %>
+<% content_for :before_content, govuk_back_link(text: "Back", href: root_path) %>
+
+<h1 class="govuk-heading-l">Nominate a school lead</h1>
+<p>This is someone who is responsible for running induction at your school.</p>
+<p>They will:</p>
+<ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-8">
+  <li>confirm the type of induction programme your school wants to run</li>
+  <li>tell us which early career teachers and mentors are starting a programme</li>
+  <li>receive any future notifications about this service</li>
```

Comment:
should these have semicolons?

Reply:
https://miro.com/app/board/o9J_ldVNkCY=/?moveToWidget=3074457355260567162&cot=14 (changed recently so copy is different) doesn't have them, neither does the prototype. Soo, probably not. 

And the only semicolons I see in erb files are some naughty inline styles :D

Reply:
https://www.imperial.ac.uk/brand-style-guide/writing/grammar/punctuation/bullet-points/

Reply:
That of course it's just imperial unis internal brand style guide, but it's a pretty standard convention

Reply:
The government disagrees https://design-system.service.gov.uk/styles/typography/list-spaced/index.html

Reply:
From https://design-system.service.gov.uk/styles/typography/#adding-extra-spacing-between-list-items:~:text=Adding%20extra%20spacing%20between%20list%20items

Reply:
yep you're right, plus https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#bullet-points-steps